addons/hr_recruitment/views/hr_applicant_views.xml

```diff
@@ -313,7 +313,7 @@
         <field name="name">Hr Applicants kanban</field>
         <field name="model">hr.applicant</field>
         <field name="arch" type="xml">
-            <kanban default_group_by="stage_id" class="o_kanban_applicant" quick_create_view="hr_recruitment.quick_create_applicant_form" sample="1">
+            <kanban default_group_by="stage_id" class="o_kanban_applicant o_search_matching_applicant" quick_create_view="hr_recruitment.quick_create_applicant_form" sample="1">
```

Comment:
It's on me for the comment in `hr_recruitment_skills/views/hr_applicant_views.xml`, it should have been here :bowing_man: 
I have the action "search matching applicants" in the "all applications" menu when switching the view to the kanban one:
![image](https://github.com/odoo/odoo/assets/71561613/eaaf5390-6388-4032-a0eb-16d34f86e094)

To be more extensive, this option is available in both kanban views (in "all applications" and when coming from a specific job) but available in neither list views.

However, as I understand the spec, the action should be both in tree and kanban view when coming from a job, but neither in the list nor the kanban when going in "all application" :) That might require to have two additional primary view in `hr_recruitment` to add that class and use those views in the actions to see applicants going through a job. Maybe it could also just be done by context?

Reply:
Ohh right all those views are common for both actions that's why both will have that search option
I've added the context to differentiate the action
Now it won't show in all application